### DISCUSSION TITLE
Why does a date appear with new entries in todo.txt?

### DISCUSSION THREAD

**Discussion by Marxsal** [gsantner/markor#2122] [Q&A] [answered]
When you make a carriage return in To-Do, each line is preceded by a date. I don't think this matches any work flow that I'm aware of, and each time I would need to delete the date. Is there a way to turn off this behaviour ?

I went through the settings for todo.txt, but didn't see anything that conformed to this behaviour. It feels like a bug, but maybe it's useful to someone else. 

Thanks!

**Comment by gsantner** [comment]
This is usually how todo.txt tasks are defined, starting with the creation date.

Please take a look at the todo.txt format, also referenced in a few documentation files in this project.

**Reply by Marxsal** [reply]
I did look at the docs for todo.txt. They suggested a date stamp as optional, not mandatory. The examples provided showed list items mostly without dates, and when there were dates it was because there was also a completion date.

**Reply by gsantner** [reply]
It's optional, but usually it's added in todo.txt applications. Feel free to delete the date if you don't need it. 

It's somewhat the minimum thing you should give tasks as metadata though 😄 .

**Reply by Marxsal** [reply]
> It's somewhat the minimum thing you should give tasks as metadata though 

Actually it's not. Per the book, in the GTD method, you only use dates for actual events that occur on dates (like meetings or appointments). Not for postponing or scheduling a task. But also, you don't use priorities in GTD either.

**Comment by harshad1** [comment, marked as answer]
Disabling auto-format removes this feature. 3 dots -> file settings

**Reply by Marxsal** [reply]
I guess this will do for now. Thanks for the suggestion!

**Comment by KeronCyst** [comment]
I, too, would like the ability to use todo.txt formatting without the task date insertion, which is meaningless to me. Could a checkbox be added to toggle this off somewhere?
